internal/api/openapi/openapi.json

```diff
@@ -0,0 +1,294 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "ROS OCP Backend API",
+    "version": "1.0.0"
+  },
+  "paths": {
+    "/api/cost-management/v1/recommendations/openshift/": {
```

Comment:
```suggestion
    "/api/cost-management/v1/recommendations/openshift": {
```

Reply:
@kgaikwad  The trailing slashes are present on the endpoint routes in the ROS OCP Architecture document, I have requested for clarification [above](https://github.com/RedHatInsights/ros-ocp-backend/pull/18#discussion_r1139796661)

Reply:
Done, changes pushed